tests/common-tests.h

```diff
@@ -317,6 +330,9 @@ class test_context_base
     // to be taken into account (currently this is only the case for Firebird).
     virtual void on_after_ddl(session&) const { }
 
+    // Override this if the backend does not support trailing space character type
+    virtual bool has_trailing_spaces_in_char_type(session&) const { return true; }
```

Comment:
I think the function name is confusing, it doesn't check if the backend uses trailing spaces, as `has_` prefix implies, but needs to be called in order to enable, or at least try to enable, standard `CHAR` handling. So I'd call it something like `enable_std_char_padding()`, with the understanding that it returns false if this can't be done (SQLite).